### PR TITLE
EN-2030 Fixed labels in seo settings section (master)

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by eugeniosant** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by eugeniosant** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java [hunk 1]
```diff
@@ -154,65 +151,58 @@ protected String readFile(String path, PageSettingsAction action) {
...
+                if (null != is) {
+                    this.getStorageManager().saveFile(ROBOT_FILENAME, false, is);
+                } else {
+                    this.getStorageManager().deleteFile(ROBOT_FILENAME, false);
                 }
-                String xmlParams = this.getConfigManager().getConfigItem(SystemConstants.CONFIG_ITEM_PARAMS);
-                Map<String, String> systemParams = SystemParamsUtils.getParams(xmlParams);
-                if (!StringUtils.isEmpty(alternativePath)) {
-                    systemParams.put(JpseoSystemConstants.ROBOT_ALTERNATIVE_PATH_PARAM_NAME, alternativePath);
+            } else if (!PageSettingsUtils.isRightPath(alternativePath)) {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java]: Is there a test for this already? Seems like a lot of logic

File: plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java [hunk 2]
```diff
@@ -154,65 +151,58 @@ protected String readFile(String path, PageSettingsAction action) {
...
+                            file.delete();
+                        }
+                    } catch (Exception e) {
+                        logger.error("error deleting file {}", alternativePath);
+                    }
                 }
-                String newXmlParams = SystemParamsUtils.getNewXmlParams(xmlParams, systemParams, true);
-                this.getConfigManager().updateConfigItem(SystemConstants.CONFIG_ITEM_PARAMS, newXmlParams);
             }
+            String xmlParams = this.getConfigManager().getConfigItem(SystemConstants.CONFIG_ITEM_PARAMS);
```
Comment (joewhite101) [inline plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java]: Needs test?

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java (hunk 1 above)
Comment (eugeniosant) [reply plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java]: Done

File: plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java (hunk 2 above)
Comment (eugeniosant) [reply plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageSettingsActionAspect.java]: Done

**Comment by entando-service** [comment]
All is Well

**Pull request merged**